Purchases/Public/RevenueCat.h

```diff
@@ -25,32 +25,7 @@ FOUNDATION_EXPORT const unsigned char RevenueCatVersionString[];
 
 NS_ASSUME_NONNULL_BEGIN
```

Comment:
should we just get rid of this file? Is it needed?

Reply:
💯, deleted.